duckduckgo/iOS
# Add to Home Row experiment #3

Task/Issue URL: 
- https://app.asana.com/0/856498667320406/1154159749265353
- https://app.asana.com/0/414235014887631/1156443180176857

**Description**:
Added:
- Onboarding flow for Add to Home Row CTA (with video).
- Custom alert flow for Add to Home Row CTA (with video).
- Pixel in case WebKit process terminates.

**Steps to test this PR**:

All flows:
- Onboarding screen should look good after layout and color tweaks.
- set_version script should work after recent tweaks.
- Home Row reminder should be shown after day 3 in case video is not played to the end and user does not visit Instructions in settings.

Control variant:
- Ensure unified CTA (with video) is displayed after onboarding within the Home Screen (inside Tab) and it looks good.

Alert variant:
- Ensure Alert-style CTA (with video) is displayed after onboarding above Main Screen.

Onboarding variant:
- Ensure 2nd onboarding screen is displayed instead of CTA.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
